src/main/java/model/figure/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package model.figure;
+
+import model.Figure;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+
+    public Rectangle(String color, int firstSide, int secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public int calculateArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: Rectangle, area:  " + calculateArea()
+                + " sq. units, first side: " + firstSide + "first side: "
```

Comment:
There is a repeated text in the print statement. It should be 'second side' instead of repeating 'first side'.